components/brave_new_tab_ui/components/default/braveToday/cards/cardIntro.tsx

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+
+// Feature-specific components
+import * as Card from '../cardIntro'
+import BraveTodayLogo from '../braveTodayLogo.svg'
+import { getLocale } from '../../../../../common/locale'
+
+class HeaderBlock extends React.PureComponent<{}, {}> {
+  onClickHideWidget = () => {
```

Comment:
is this being used?

Reply:
Huh... I didn't notice that. Not my code, but assume the plan was to be able to remove the intro card. Oh well, it's not hideable for this PR :-) Removed.